Setup-Docs.md

```diff
@@ -0,0 +1,11 @@
+# Getting started
```

Comment:
I don't think this file is necessary. All this is covered in in the README... Which I'll update one day. The exception is that the README doesn't talk about swarm as a dependency. This is in part because it comes with Docker, we just have to initialize it.